examples/bridge-app/linux/main.cpp

```diff
@@ -257,8 +260,15 @@ DataVersion gComposedTempSensor2DataVersions[MATTER_ARRAY_SIZE(bridgedTempSensor
 
 // ---------------------------------------------------------------------------
 
+#if !CHIP_CONFIG_USE_ENDPOINT_UNIQUE_ID
 int AddDeviceEndpoint(Device * dev, EmberAfEndpointType * ep, const Span<const EmberAfDeviceType> & deviceTypeList,
                       const Span<DataVersion> & dataVersionStorage, chip::EndpointId parentEndpointId = chip::kInvalidEndpointId)
+#else
+
+int AddDeviceEndpoint(Device * dev, EmberAfEndpointType * ep, const Span<const EmberAfDeviceType> & deviceTypeList,
+                      const Span<DataVersion> & dataVersionStorage, chip::CharSpan epUniqueId,
+                      chip::EndpointId parentEndpointId = chip::kInvalidEndpointId)
+#endif
```

Comment:
```suggestion
int AddDeviceEndpoint(Device * dev, EmberAfEndpointType * ep, const Span<const EmberAfDeviceType> & deviceTypeList,
                      const Span<DataVersion> & dataVersionStorage, 
#if !CHIP_CONFIG_USE_ENDPOINT_UNIQUE_ID                      
                      chip::CharSpan epUniqueId,
#endif                      
                      chip::EndpointId parentEndpointId = chip::kInvalidEndpointId)
```

Seems to make it more obvious about "what is different". 
It also changes a method with 5 parameters to 6 parameters, we may opt to provide some `Params` structure instead, in which case we would ifdef a member of that struct and it would be easier to read.

Reply:
You meant 
`#if CHIP_CONFIG_USE_ENDPOINT_UNIQUE_ID                      
                      chip::CharSpan epUniqueId,
#endif `

Reply:
yes, copy and paste error. Will edit